Packages/install_redeem

```diff
@@ -10,12 +10,27 @@ cd Packages
 
 echo "**install_redeem**"
 set -x
+
+apt-get install --no-install-recommends -y \
+swig \
+python-scipy \
+python-numpy \
+python-smbus \
+ti-pru-cgt-installer
+
+if [ ! -d "/usr/src/pru-software-support-package" ]; then
+  git clone --branch v5.1.0 --single-branch --depth 1 git://git.ti.com/pru-software-support-package/pru-software-support-package.git /usr/src/pru-software-support-package
+fi
+
+pip install numpy evdev spidev Adafruit_BBIO Adafruit-GPIO sympy docutils sh
```

Comment:
setup.py is supposed to handle the pip installs that are in the venv.

Reply:
Currently building with this removed to make sure Redeem still starts in the resulting image.

Reply:
I agree that this line should be remove-able, but it currently breaks redeem startup:
```
Jan 10 18:33:56 kamikaze systemd[1]: Started The Replicape Daemon.
Jan 10 18:34:24 kamikaze python2[776]: Adafruit_BBIO: version <unknown> initialized
Jan 10 18:34:24 kamikaze redeem[769]: Traceback (most recent call last):
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/local/./bin/replicape_controller", line 11, in <module>
Jan 10 18:34:24 kamikaze redeem[769]:     load_entry_point('Redeem', 'console_scripts', 'redeem')()
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 480, in load_entry_point
Jan 10 18:34:24 kamikaze redeem[769]:     return get_distribution(dist).load_entry_point(group, name)
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2693, in load_entry_point
Jan 10 18:34:24 kamikaze redeem[769]:     return ep.load()
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2324, in load
Jan 10 18:34:24 kamikaze redeem[769]:     return self.resolve()
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2330, in resolve
Jan 10 18:34:24 kamikaze redeem[769]:     module = __import__(self.module_name, fromlist=['__name__'], level=0)
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/local/src/redeem/redeem/Redeem.py", line 37, in <module>
Jan 10 18:34:24 kamikaze redeem[769]:     from Stepper import *
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/local/src/redeem/redeem/Stepper.py", line 32, in <module>
Jan 10 18:34:24 kamikaze redeem[769]:     from Key_pin import Key_pin
Jan 10 18:34:24 kamikaze redeem[769]:   File "/usr/local/src/redeem/redeem/Key_pin.py", line 31, in <module>
Jan 10 18:34:24 kamikaze redeem[769]:     from evdev import *
Jan 10 18:34:24 kamikaze redeem[769]: ImportError: No module named evdev
Jan 10 18:34:25 kamikaze systemd[1]: redeem.service: Main process exited, code=exited, status=1/FAILURE
Jan 10 18:34:25 kamikaze systemd[1]: redeem.service: Failed with result 'exit-code'.
```

Would you object if we left it as-is for now and fixed it in a followup?

Reply:
You really should merge the change (PR#86) that I had approved before your PR was finished. That way you would not need this horrible hack.

Reply:
It should be, but right now removing this still breaks the image. Unless there's an urgent reason, I think it should be fine to keep the status quo and fix this in a followup?

(I didn't add this line myself - it just moved from install_dependencies to install_redeem)

Reply:
That all depends on which version of Redeem you are loading. In any case, I think that we should not clutter the history of this file and leave it to the Redeem repository to take care of whatever it needs.

Besides, I think that with the new virtual environments, unlike modules loaded by apt-get, modules loaded by pip in the system space will not be available to the intended program.

Reply:
Try adding `python-evdev` to the apt-get dependencies... `evdev` itself doesn't exist.